src/core/services/theming/theming.js

```diff
@@ -282,6 +357,8 @@ function ThemingProvider($mdColorPalette) {
       alwaysWatchTheme = alwaysWatch;
     },
 
+    enableBrowserColor: enableBrowserColor,
+
```

Comment:
Oh so you we're suggesting that we would enable them the option of setting what ever color they want?
I'm against it, in that case i'd say to them, use it manually.